src/config/common.appsettings.json

```diff
@@ -17,6 +17,9 @@
       "Authority": "https://login.microsoftonline.com/61c2b580-8619-4baf-8040-bac4442e29a6",
       "ClientId": "",
       "ValidateAuthority": true
+    },
+    "AzureKeyVault": {
```

Comment:
```suggestion
    "KeyVault": {
```

Reply:
Altered the name here and its reference in program.cs